src/nwb_benchmarks/core/_base_benchmark.py

```diff
@@ -27,19 +31,43 @@ class BaseBenchmark:
 
     parameter_cases: Union[Dict[str, Dict[str, Any]], None] = None
 
-    def __init__(self):
+    def __new__(cls, *args, **kwargs) -> Self:
+        instance = super().__new__(cls)
+
         # Unpack parameter cases dictionary into ASV expected form
-        if self.parameter_cases is not None:
-            self.param_names = list(next(iter(self.parameter_cases.values())).keys())
-            self.params = (
+        if cls.parameter_cases is not None:
+            cls.param_names = list(next(iter(cls.parameter_cases.values())).keys())
+            cls.params = (
                 [
-                    [parameter_case[param_name] for parameter_case in self.parameter_cases.values()]
-                    for param_name in self.param_names
+                    [parameter_case[param_name] for parameter_case in cls.parameter_cases.values()]
+                    for param_name in cls.param_names
                 ]
-                if len(self.param_names) > 1
+                if len(cls.param_names) > 1
                 else [
                     parameter
-                    for parameter_case in self.parameter_cases.values()
+                    for parameter_case in cls.parameter_cases.values()
                     for parameter in parameter_case.values()
                 ]
             )
+
+            # ASV automatically forms a cartesian product over all params
```

Comment:
Or, I guess I had already pushed the fix to the params thing on the other branch. Now #63 has a minor residual fix to parsing the results from the corrected intermediate results files as well as some more results